nautobot/docs/models/extras/configcontext.md

```diff
@@ -13,6 +13,7 @@ Sometimes it is desirable to associate additional data with a group of devices o
 * Tenant
 * Device redundancy group
 * Tag
+* Dynamic Group
```

Comment:
Let's indicate that this is added in a specific release. We actually should have done the same for device redundancy group.

```suggestion
* Tenant
* Tag

+++ 1.5.0
    * Device redundancy group

+++ 1.5.10
    * Dynamic group
```